gapic/src/main/com/google/gapid/perfetto/views/TrackContainer.java

```diff
@@ -52,20 +52,40 @@
   private TrackContainer() {
   }
 
+  public static enum Truncate {
+    Left {
```

Comment:
I thought about more, this really should be LEFT and RIGHT.  This is an enum, so it should be named as such.

As a side note, the fact that we have to discuss how to name this partly shows why I think this way of doing this is more confusing than it needs to be.

Reply:
https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s4.8.1-enum-classes

Reply:
I gave it some more thought and decided to do away with enums, since importing stuff from RenderContext into models is not something we have done anywhere else.